### PR TITLE
explain multi-project quota

### PR DESCRIPTION
**Pull request by deads2k**
Adds documentation that explains how to use https://trello.com/c/qJUVjQnj/713-13-multi-project-quota

### CODE REVIEW

File: admin_guide/multiproject_quota.adoc [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+A multi-project quota, defined by a `*ClusterResourceQuota*` object, allows xref:../admin_guide/quota.adoc#admin-guide-quota[quota]
+to be shared across multiple projects.  Resources used in each selected project
+will be aggregated and that aggregate will be used to limit resources across
+all the selected projects.
+
+=== Selecting Projects
+
+Projects can be selected based on either annotation selection, label selection, or both.
+For example, `oc create clusterquota for-user --project-annotation-selector openshift.io/requester=deads --hard pods=10 --hard secrets=20`
```
Comment (mfojtik) [inline admin_guide/multiproject_quota.adoc]: your name will live in docs forever :-)

**Comment by deads2k** [comment]
@adellape @ahardin-rh Is there a tag I should use for 3.3 items? I've got a few doc pulls that should go in.

File: admin_guide/multiproject_quota.adoc [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+toc::[]
+
+== Overview
+
+A multi-project quota, defined by a `*ClusterResourceQuota*` object, allows xref:../admin_guide/quota.adoc#admin-guide-quota[quota]
+to be shared across multiple projects.  Resources used in each selected project
+will be aggregated and that aggregate will be used to limit resources across
+all the selected projects.
+
```
Comment (ahardin-rh) [inline admin_guide/multiproject_quota.adoc]: We need to add an anchor just abovethe heading:
[[multi-project-quotas-selecting-projects]]

File: admin_guide/multiproject_quota.adoc [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+<2> A simple key/value selector for annotations
+<3> A label selector that can be used to select projects
+<4> A per-namespace map that describes current quota usage in each selected project
+<5> The aggregate usage across all selected projects
+====
+
+This multi-project quota document will control all projects requested by "deads" using the default
+project request endpoint.  He will be limited to ten pods and twenty secrets.
+
+
```
Comment (ahardin-rh) [inline admin_guide/multiproject_quota.adoc]: We need to add an anchor just above the heading:
[[multi-project-quotas-viewing-applicable-clusterresourcequotas]]

File: admin_guide/multiproject_quota.adoc [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+Annotations:  <none>
+Label Selector: <null>
+AnnotationSelector: map[openshift.io/requester:deads]
+Resource  Used  Hard
+--------  ----  ----
+pods    1 10
+secrets   9 20
+----
+
+
```
Comment (ahardin-rh) [inline admin_guide/multiproject_quota.adoc]: We need to add an anchor just above the heading:
[[multi-project-quotas-selection-granularity]]

File: admin_guide/multiproject_quota.adoc (hunk 1 above)
Comment (ahardin-rh) [inline admin_guide/multiproject_quota.adoc]: We tend to use generic user names, so I am going to unfortunately be a killjoy and recommend replacing instances of "deads" with a replaceable <user-name>  :crying_cat_face:

File: admin_guide/multiproject_quota.adoc [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+This multi-project quota document will control all projects requested by "deads" using the default
+project request endpoint.  He will be limited to ten pods and twenty secrets.
+
+
+=== Viewing Applicable ClusterResourceQuotas
+
+A project-admin is not allowed to create or modify the multi-project quota that limits
+him, but he is allowed to view the multi-project quota documents are applied to his project.
+He can do this via the `AppliedClusterResourceQuota` resource.
```
Comment (ahardin-rh) [inline admin_guide/multiproject_quota.adoc]: `*AppliedClusterResourceQuota*`

File: admin_guide/multiproject_quota.adoc [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+`oc describe AppliedClusterResourceQuota` produces
```
Comment (ahardin-rh) [inline admin_guide/multiproject_quota.adoc]: colon after produces

File: admin_guide/multiproject_quota.adoc [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+A multi-project quota, defined by a `*ClusterResourceQuota*` object, allows xref:../admin_guide/quota.adoc#admin-guide-quota[quota]
+to be shared across multiple projects.  Resources used in each selected project
+will be aggregated and that aggregate will be used to limit resources across
+all the selected projects.
+
+=== Selecting Projects
+
+Projects can be selected based on either annotation selection, label selection, or both.
+For example, `oc create clusterquota for-user --project-annotation-selector openshift.io/requester=deads --hard pods=10 --hard secrets=20`
+creates
```
Comment (ahardin-rh) [inline admin_guide/multiproject_quota.adoc]: colon after creates

**Comment by ahardin-rh** [comment]
@deads2k I applied labels/milestone for forthcoming 3.3 and added a few comments. Cheers! :beers:

File: admin_guide/multiproject_quota.adoc (hunk 2 above)
Comment (deads2k) [inline admin_guide/multiproject_quota.adoc]: > We need to add an anchor just above the heading:
> [[multi-project-quotas-selecting-projects]]

done

**Comment by deads2k** [comment]
comments addressed.

**Comment by ahardin-rh** [comment]
@deads2k Thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/multiproject_quota.adoc#admin-guide-muliproject-quota[Setting Multi-Project Quotas]
|New topic about how to set multi-project quotas.
%

File: admin_guide/multiproject_quota.adoc [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+====
+
+
+[[multi-project-quotas-selection-granularity]]
+=== Selection Granularity
+
+Because of the locking consideration when claiming quota allocations, the 
+number of active projects selected by a multi-project quota is an important consideration.
+Selecting more than 100 projects under a single multi-project quota may have
+deleterious effects on API server responsiveness in those projects.
```
Comment (derekwaynecarr) [inline admin_guide/multiproject_quota.adoc]: deterious?

File: admin_guide/multiproject_quota.adoc [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+This multi-project quota document will control all projects requested by `"<user-name>"` using the default
+project request endpoint.  He will be limited to ten pods and twenty secrets.
+
+
+[[multi-project-quotas-viewing-applicable-clusterresourcequotas]]
+=== Viewing Applicable ClusterResourceQuotas
+
+A project-admin is not allowed to create or modify the multi-project quota that limits
+him, but he is allowed to view the multi-project quota documents are applied to his project.
+He can do this via the `*AppliedClusterResourceQuota*` resource.
```
Comment (derekwaynecarr) [inline admin_guide/multiproject_quota.adoc]: write doc gender neutral

**Review by derekwaynecarr — COMMENTED** [review]
two comments, then lgtm

**Pull request merged**